.gitignore

```diff
@@ -0,0 +1,3 @@
+TAGS
+dist-newstyle/*
+dist/*
```

Comment:
could you add this to your `gitignore_global` instead?

Reply:
Ok, that works.